components/list/test/list-item-drag-handle.visual-diff.js

```diff
@@ -34,14 +41,14 @@ describe('d2l-list-item-drag-handle', () => {
 			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
 		});
 
-		it('keyboard-mode', async function() {
+		itWithReload('keyboard-mode', async function() {
```

Comment:
Interesting... what was happening here? Was this always kinda flakey?

Reply:
Just another retry issue - first attempt is fine, but retries are different. Since there's an intended diff here, it was retrying and ending up wrong (tooltip was no longer displaying).

I created the `itWithReload` function to just do a page reload after each attempt, I could have added `page.reload()` on an afterEach for all the tests, or stuck this in its own describe block and done an afterEach there, but this seemed cleaner.

Reply:
Also, there might have been a less nuclear option than `page.reload()` to get retries to work, but I didn't want to dig into the specifics of what would and wouldn't put it back into a "clean" state 😄.

Reply:
I think that keyboard tooltip only gets displayed once per page load per list, or once per page load, so the reload is probably the safest.